### DISCUSSION TITLE
Question about preloading

### DISCUSSION THREAD

**Discussion by barredo** [vidstack/player#876] [Q&A]
Hello. I'm finishing up implementing the community skin, and I was wondering If i could have this settings:

- do not preload anything from the file, even the metadata
- pass the duration of the media file to the player (like in plyr)

can it be done with the current attributes and settings? do I need some custom function? 

Thank you!

**Comment by mihar-22** [comment]
Hey @barredo!

The [player has loading strategies](https://www.vidstack.io/docs/player/core-concepts/loading#loading-strategies) which defines when content should be loaded. Nothing is loaded (including metadata) until the specified event occurs. The custom loading strategy might be what you're looking for. 

There is currently no way to set the duration yourself but we can explore adding that as a feature in a separate issue.
